indico/modules/events/tracks/operations.py

```diff
@@ -31,4 +32,21 @@ def create_track(event, data):
     logger.info('Track %r created by %s', track, session.user)
     event.log(EventLogRealm.management, EventLogKind.positive, 'Tracks',
               'Track "{}" has been created.'.format(track.title), session.user)
+    flash(_('Track "{}" has been created.').format(track.title), 'success')
```

Comment:
`flash` belongs in the request handler, as it is strictly related with the rendering of the page. Functions in `operations.py` could be called from `indico shell` and flashing makes no sense there.